src/psbt.cpp

```diff
@@ -329,6 +402,7 @@ bool FinalizeAndExtractPSBT(PartiallySignedTransaction& psbtx, CMutableTransacti
         result.witness.vtxinwit[i].scriptWitness = psbtx.inputs[i].final_script_witness;
         PSBTInput& input = psbtx.inputs[i];
 
+        /*
```

Comment:
Commented-out (and not removed) code without any further comment with justification of not removing it can be confusing and distracting. Is this commented out just temporary ?

Reply:
Yes, it is just temporary. The comment is to just make the code compile so that the action that this code does can be reimplemented later in this PR.